service/debugger/debugger.go

```diff
@@ -1261,6 +1261,12 @@ func (d *Debugger) GetVersion(out *api.GetVersionOut) error {
 			out.Backend = d.config.Backend
 		}
 	}
+
+	out.TargetGoVersion = d.target.BinInfo().Producer()
+
+	out.MinSupportedVersionOfGo = fmt.Sprintf("1.%d.0", goversion.MinSupportedVersionOfGoMinor)
```

Comment:
We should make the major version a const as well just for consistency and so we don't have to change it later.